Sources/ApodiniDeployRuntimeSupport/ExportStructureCommand.swift

```diff
@@ -0,0 +1,86 @@
+//
+// This source file is part of the Apodini open source project
+//
+// SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import ArgumentParser
+import Apodini
+import Runtime
+
+/// A CLI command responsible for exporting the structure of the web service.
+/// The export can be configured by providing options and arguments.
+/// The default implementation of this command exports the structure without any constraints.
+/// A deployment provider may define a subcommand to this to account for custom options.
+/// **Note**: Should not be called by the user directly
+public struct ExportStructureCommand: ParsableCommand {
+    /// Often used options that can be imported into a subcommand.
+    public struct ExportOptions: ParsableArguments {
+        @Argument(help: "The location of the json file")
+        public var filePath: String = "service-structure.json"
+        
+        @Option(help: "The identifier of the deployment provider")
+        public var identifier: String = "de.lukaskollmer.ApodiniDeploymentProvider.Localhost"
```

Comment:
Localhost doesn't seem correct here. I would go for a more neutral `de.tum.in.ase.apodini.deploy` or something like this.